File: C++/C++Basic/06.Conditional-Statements-Advanced-Exercise/07.hotelRoom.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string month;
    int nights;

    cin >> month;
    cin >> nights;

    double studioTotalCost = 0.0;
    double apartmentTotalCost = 0.0;

    if (month == "May" || month == "October")
    {
        studioTotalCost = nights * 50;
        apartmentTotalCost = nights * 65;

        if (nights > 14)
        {
            studioTotalCost *= 0.7;
        }
        else if (nights > 7)
        {
            studioTotalCost *= 0.95;
        }
    }
    else if (month == "June" || month == "September")
    {
        studioTotalCost = nights * 75.20;
        apartmentTotalCost = nights * 68.70;
        
        if (nights > 14)
        {
            studioTotalCost *= 0.80;
        }
    }
    else if (month == "July" || month == "August")
    {
            studioTotalCost = nights * 76;
            apartmentTotalCost = nights * 77;
    }

    if (nights > 14)
    {
        apartmentTotalCost *= 0.90;
    }

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << "Apartment: " << apartmentTotalCost <<" lv." << endl;
    cout <<  "Studio: " << studioTotalCost << " lv." << endl;

    return 0;
}

File: Python/Basic/06.Conditional-Statements-Advanced-Exercise/07.hotel_room.py
months = input()
count_night = int(input())

price_studio = 0
price_apartment = 0

if months == "May" or months == "October":
    price_studio = 50 * count_night
    price_apartment = 65 * count_night

    if count_night > 14:
        price_studio *= 0.7
        price_apartment *= 0.90
    elif count_night > 7:
        price_studio *= 0.95

elif months == "June" or months == "September":
    price_studio = 75.20 * count_night
    price_apartment = 68.70 * count_night

    if count_night > 14:
        price_studio *= 0.80
        price_apartment *= 0.90

elif months == "July" or months == "August":
    price_studio = 76 * count_night
    price_apartment = 77 * count_night

    if count_night > 14:
        price_apartment *= 0.90

print(f"Apartment: {price_apartment:.2f} lv.")
print(f"Studio: {price_studio:.2f} lv.")